vars/deploy-on-openstack.yml

```diff
@@ -30,6 +30,14 @@ deploy_on_openstack_osh_repos_per_imagename:
     - openSUSE-Leap-15.0-Cloud
     - Leap15develtools
     - socok8s
+  SLE-15-SP1-JeOS-GM:
```

Comment:
Could you add a ```#TODO: remove SLES-15-SP1-openstack-master when caasp3 compat is removed```, please?

Reply:
@evrardjp I don't see it in the list of repositories, why should I add a comment about a repository that isn't used? ;) I guess I should squad the commits at some point..

Reply:
That was just before you pushed a patch I think or because I am blind. Just ignore this. :)